profiles/common/modules/custom/tmgmt_poetry/src/tmgmt_poetry.behat.inc

```diff
@@ -148,12 +148,14 @@ class TmgmtPoetrySubContext extends RawDrupalContext implements DrupalSubContext
 
 
   /**
-   * Get the Current Node Id in the URL.
+   * Get the Current Node Id.
    */
   private function getCurrentNodeId() {
-    $url = $this->getSession()->getCurrentUrl();
-    preg_match_all('/node\/(\d*)/', $url, $matches);
-    return (isset($matches[1][0]) ? $matches[1][0] : FALSE);
```

Comment:
Agreed. Should be for instance getLastCreatedNodeId().
Also, are we sure we will always get the id we expect (in this ticket context I mean) ?